tasks/dataset-split/task.yaml

```diff
@@ -0,0 +1,18 @@
+id: dataset-split
+name: Dataset Split
+description: Task to split a dataset into a user-defined number of parts
+is_active: true
+project_type: 
+param_groups:
+  - name: inputs
+    params:
+      - name: dataset
+        value: null
+        required: true
+        data_type: dataset
+        description: User-entered dataset
```

Comment:
A better description might me "Dataset which is going to be split into multiple parts"